accumulo-store/src/main/java/gaffer/accumulostore/retriever/AccumuloItemRetriever.java

```diff
@@ -79,12 +80,16 @@ protected ElementIterator(final Iterator<? extends SEED_TYPE> idIterator) throws
             idsIterator = idIterator;
             count = 0;
             final Set<Range> ranges = new HashSet<>();
-            while (idsIterator.hasNext() && count < store.getProperties().getMaxEntriesForBatchScanner()) {
-                count++;
-                try {
-                    addToRanges(idsIterator.next(), ranges);
-                } catch (final RangeFactoryException e) {
-                    LOGGER.error("Failed to create a range from given seed pair", e);
+            if (operation instanceof GetAllElements) {
+                ranges.add(new Range());
```

Comment:
I think possibly a new Retriever subclass is needed for all we may be able to simplify other things in a dedicated reliever over doing this check here, this also has a problem of tying in a specific operation to this functionality as opposed to operation calling specific retrievers in their handler.